app/src/main/res/layout/activity_create_config.xml

```diff
@@ -94,7 +94,7 @@
                         android:layout_width="@dimen/width_zero"
                         android:layout_height="wrap_content"
                         android:layout_gravity="center"
-                        android:layout_weight="@dimen/weight_one" />
+                        android:layout_weight="@dimen/weight_three" />
```

Comment:
Perhaps, instead of updating the `layout_weight`, we could migrate the parent `LinearLayout` to a `ConstraintLayout`, which would better support a large number of screen sizes.